docs/get-started/cardano-node/running-cardano.md

```diff
@@ -124,22 +103,43 @@ To run `cardano-node` you enter something like this into the terminal:
 To get the complete list of available options, use `cardano-node run --help`
 
 ```
-Usage: cardano-node run [--topology FILEPATH] [--database-path FILEPATH] 
-                        [--socket-path FILEPATH] 
-                        [--byron-delegation-certificate FILEPATH] 
-                        [--byron-signing-key FILEPATH] 
-                        [--shelley-kes-key FILEPATH] 
-                        [--shelley-vrf-key FILEPATH] 
-                        [--shelley-operational-certificate FILEPATH] 
-                        [--bulk-credentials-file FILEPATH] [--host-addr IPV4] 
-                        [--host-ipv6-addr IPV6] [--port PORT] 
-                        [--config NODE-CONFIGURATION] [--validate-db]
+Usage: cardano-node.exe run [--topology FILEPATH]
```

Comment:
cardano-node (instead of .exe reference on windows) would still be preferable IMO

Reply:
Ah indeed good catch